data/preprocessor.js

```diff
@@ -0,0 +1,22 @@
+import nodeLogger from '../src/app/helpers/logger.node';
+import applyTimestampRules from './rules/timestamp';
+
+export default (jsonRaw = {}, rules = [applyTimestampRules]) => {
```

Comment:
While i love the rules, we shouldnt use it as a default. Either you pass in rules via the function, or theyre set in an array inside it. we should do the latter imo

Reply:
Why not? DRY principle - we set it in one place within the `data/` folder, no need to edit the application when we apply business rule block changes in the preprocessor. And being able to pass it in as a parameter means we can test it 👍

Reply:
what wil happen when we have more? just gunna keep stringing them togeather in the function default?

Reply:
Yep, or could have a `const defaultRules = [...]` above the export, and set `rules = defaultRules` in the param. That's probably nicest - will do that now.